floodsub.go

```diff
@@ -16,7 +16,11 @@ import (
 	timecache "github.com/whyrusleeping/timecache"
 )
 
-const ID = protocol.ID("/floodsub/1.0.0")
+const (
+	ID                     = protocol.ID("/floodsub/1.0.0")
+	defaultMaxConcurrency  = 10
```

Comment:
Let's make this a 100 at least (or even 1000) -- it's a global throttle on all messages not just per channel.